lib/query/querybuilder.js

```diff
@@ -269,6 +269,11 @@ class Builder extends EventEmitter {
     return this;
   }
 
+  using(tables) {
```

Comment:
what if someone tries using it outside of pg?

Reply:
Nothing. I dont see how I can restrict function access in query builder. Or maybe I put a warning ?

Reply:
I would probably make default implementation throw an error, and override it for pg.

Reply:
Oh, yes.... I will make that.